projects/swimlane/ngx-ui/src/lib/components/calendar/calendar.component.ts

```diff
@@ -550,6 +696,15 @@ export class CalendarComponent implements OnInit, AfterViewInit, ControlValueAcc
     this.cdr.detectChanges();
   }
 
+  formatDate(date: Date): string {
+    const months: string[] = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+    const month: string = months[date.getMonth()];
+    const day: number = date.getDate();
+    const year: number = date.getFullYear();
+
+    return `${month} ${day}, ${year}`;
```

Comment:
How will this work for i18n? Is there a method from `moment` we can leverage to customize the visible date based on locale?

Reply:
Changed Accordingly

Reply:
@steveblue What should be the changes in Change log 
I mean the version that I should use